<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgEditor</class>
 <widget class="QWidget" name="wdgEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">wdgEditor</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lblStatistics">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>9</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string notr="true">lblStatistics</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6"/>
     </item>
     <item>
      <widget class="QFrame" name="fraSearch">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="ledSearch">
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="maxLength">
           <number>32767</number>
          </property>
          <property name="placeholderText">
           <string>Font or character</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCloseSearch">
          <property name="icon">
           <iconset theme="application-exit"/>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTreeView" name="tvwChars">
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="btnEditorPreferences">
         <property name="icon">
          <iconset theme="document-properties"/>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnMoveUp">
         <property name="icon">
          <iconset theme="go-up"/>
         </property>
         <property name="autoRepeat">
          <bool>true</bool>
         </property>
         <property name="autoRepeatInterval">
          <number>25</number>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnMoveDown">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="icon">
          <iconset theme="go-down"/>
         </property>
         <property name="autoRepeat">
          <bool>true</bool>
         </property>
         <property name="autoRepeatInterval">
          <number>25</number>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDeleteItem">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="icon">
          <iconset theme="edit-clear"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnAddFont">
         <property name="text">
          <string>Add font</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAddCharacter">
         <property name="text">
          <string>Add character</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QStackedWidget" name="stwProperties">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="pgeFont">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="lblFontName">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>Name:</string>
             </property>
             <property name="buddy">
              <cstring>ledFontName</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="ledFontName">
             <property name="toolTip">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lblColorTexture">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>Color texture:</string>
             </property>
             <property name="buddy">
              <cstring>ledColorTexture</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="ledColorTexture">
               <property name="font">
                <font>
                 <family>Consolas</family>
                </font>
               </property>
               <property name="toolTip">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnColorTexture">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="toolTip">
                <string/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset theme="folder-open"/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lblAlphaTexture">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>Alpha texture:</string>
             </property>
             <property name="buddy">
              <cstring>ledAlphaTexture</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLineEdit" name="ledAlphaTexture">
               <property name="font">
                <font>
                 <family>Consolas</family>
                </font>
               </property>
               <property name="toolTip">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnAlphaTexture">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="toolTip">
                <string/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset theme="folder-open"/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="lblMaxHeigthOfChars">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>Maximum height of characters:</string>
             </property>
             <property name="buddy">
              <cstring>sbxMaxHeigthOfChars</cstring>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="sbxMaxHeigthOfChars">
             <property name="font">
              <font>
               <family>Consolas</family>
              </font>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::UpDownArrows</enum>
             </property>
             <property name="accelerated">
              <bool>false</bool>
             </property>
             <property name="showGroupSeparator" stdset="0">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="value">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="lblDistanceBetweenChars">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>Distance between characters:</string>
             </property>
             <property name="buddy">
              <cstring>sbxDistanceBetweenChars</cstring>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="sbxDistanceBetweenChars">
             <property name="font">
              <font>
               <family>Consolas</family>
              </font>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pgeCharacter">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item row="2" column="2" rowspan="2" colspan="2">
            <widget class="QSpinBox" name="sbxRightPixel">
             <property name="font">
              <font>
               <family>Consolas</family>
              </font>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
           <item row="0" column="2" colspan="2">
            <widget class="QLineEdit" name="ledCharacter">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <family>Consolas</family>
              </font>
             </property>
             <property name="toolTip">
              <string/>
             </property>
             <property name="maxLength">
              <number>1</number>
             </property>
             <property name="cursorMoveStyle">
              <enum>Qt::LogicalMoveStyle</enum>
             </property>
             <property name="clearButtonEnabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0" rowspan="2" colspan="2">
            <widget class="QLabel" name="lblRightPixel">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>Rightmost pixel:</string>
             </property>
             <property name="buddy">
              <cstring>sbxRightPixel</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLabel" name="lblLeftPixel">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>Leftmost pixel:</string>
             </property>
             <property name="buddy">
              <cstring>sbxLeftPixel</cstring>
             </property>
            </widget>
           </item>
           <item row="4" column="0" rowspan="2" colspan="2">
            <widget class="QLabel" name="lblHighestPixelInFontRow">
             <property name="font">
              <font>
               <family>Segoe UI</family>
              </font>
             </property>
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>Highest pixel in row:</string>
             </property>
             <property name="buddy">
              <cstring>sbxHighestPixelInFontRow</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="2" colspan="2">
            <widget class="QSpinBox" name="sbxLeftPixel">
             <property name="font">
              <font>
               <family>Consolas</family>
              </font>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
           <item row="4" column="2" rowspan="2" colspan="2">
            <widget class="QSpinBox" name="sbxHighestPixelInFontRow">
             <property name="font">
              <font>
               <family>Consolas</family>
              </font>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="lblCharacter">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>Character:</string>
             </property>
             <property name="buddy">
              <cstring>ledCharacter</cstring>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ledCharacter</tabstop>
  <tabstop>sbxLeftPixel</tabstop>
  <tabstop>sbxRightPixel</tabstop>
  <tabstop>sbxHighestPixelInFontRow</tabstop>
  <tabstop>btnMoveDown</tabstop>
  <tabstop>btnAddFont</tabstop>
  <tabstop>btnAddCharacter</tabstop>
  <tabstop>ledFontName</tabstop>
  <tabstop>ledColorTexture</tabstop>
  <tabstop>btnColorTexture</tabstop>
  <tabstop>ledAlphaTexture</tabstop>
  <tabstop>btnAlphaTexture</tabstop>
  <tabstop>sbxMaxHeigthOfChars</tabstop>
  <tabstop>sbxDistanceBetweenChars</tabstop>
  <tabstop>btnEditorPreferences</tabstop>
  <tabstop>tvwChars</tabstop>
  <tabstop>btnMoveUp</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
